apps/antalmanac/src/components/SharedRoot.tsx

```diff
@@ -83,97 +75,235 @@ const tabs: Array<TabInfo> = [
+                        </Box>
+                    }
+                    style={{ minWidth: '25%' }}
+                />
+            ))}
+        </Tabs>
+    );
+}
+
+/**
+ * For desktop, some of the tabs will be displayed on the other side.
+ * i.e. the calendar takes up the left side of the screen.
+ */
+function ScheduleManagementDesktopTabs(props: ScheduleManagementTabsProps) {
+    const { value, setActiveTab } = props;
+
+    const onChange = (_event: React.SyntheticEvent, value: number) => {
+        setActiveTab(value + 1);
+    };
+
+    return (
+        <Tabs value={value} onChange={onChange} indicatorColor="primary" variant="fullWidth" centered>
+            {scheduleManagementTabs.map((tab) => {
+                if (tab.mobile) return;
+
+                return (
+                    <Tab
+                        key={tab.label}
+                        id={tab.id}
+                        to={tab.href}
+                        component={Link}
+                        label={
+                            <div style={{ display: 'inline-flex', alignItems: 'center' }}>
+                                <tab.icon style={{ height: 16 }} />
+                                <Typography variant="body2">{tab.label}</Typography>
+                            </div>
+                        }
+                        sx={{ minHeight: 'auto', height: '44px', padding: 3, minWidth: '33%' }}
+                    />
+                );
+            })}
+        </Tabs>
+    );
+}
+
+function ScheduleManagementTabsContent(props: { activeTab: number; isMobile: boolean }) {
+    const { activeTab, isMobile } = props;
+
+    const isDark = useThemeStore((store) => store.isDark);
+
+    switch (activeTab) {
+        case 0:
+            return <Calendar isMobile={isMobile} />;
+        case 1:
+            return <CoursePane />;
+        case 2:
+            return <AddedCoursePane />;
+        case 3:
+            return (
+                <Suspense
+                    fallback={
+                        <div
+                            style={{
+                                height: '100%',
+                                width: '100%',
+                                display: 'flex',
+                                justifyContent: 'center',
+                                alignItems: 'center',
+                            }}
+                        >
+                            <img src={isDark ? darkModeLoadingGif : loadingGif} alt="Loading map" />
+                        </div>
+                    }
+                >
+                    <UCIMap />
+                </Suspense>
+            );
+
+        default:
+            return null;
+    }
 }
 
-const SharedTabs = ({ style, mobile }: SharedTabsProps) => {
+/**
+ * List of interactive tab buttons with their accompanying content.
+ * Each tab's content has functionality for managing the user's schedule.
+ */
+export default function ScheduleManagement() {
```

Comment:
nit: super naming nit, but I feel like `scheduleManagement` is not really descriptive of what's going on here with the tabs

Any reason why we can't prefer `NavigationTabs` or something that's a little closer to what the component *is*

Reply:
Neither is `SharedRoot` 🤓 

IMO, `NavigationTabs` sounds generic and similar to something you might import from a component library.

I'm not really a big fan of calling them "tabs" because that's not really what this component is. It's multiple screens that can be navigated between via tabs. i.e. It's both the tabs and the tab contents.

In the end, I think the primary purpose of this component is to allow you to manipulate your schedule through its various screens.